Source/missiles.cpp

```diff
@@ -1414,6 +1426,7 @@ void AddHorkSpawn(int mi, Point src, Point dst, int midir, int8_t /*mienemy*/, i
 void AddJester(int mi, Point src, Point dst, int midir, int8_t /*mienemy*/, int id, int /*dam*/)
 {
 	int spell = MIS_FIREBOLT;
+	auto &missile = Missiles[mi];

```

Comment:
This should be the first variable declared (move above `int spell`) since the switch statement is used to fully initialise `spell`;